src/javascripts/components/views/weaponsView.js

```diff
@@ -0,0 +1,31 @@
+import weaponsData from '../../helpers/data/weaponsData';
+import card from '../cards/weaponsCards';
+import weaponsForm from '../forms/weaponsForm';
+
+const weaponsView = () => {
+  $('#add-button').html(
+    '<button id="add-weapon" type="button" class="btn btn-dark">Add Weapons +</button>'
+  );
+  $('#add-weapon').on('click', weaponsForm.weaponForm);
+
+  $('#app').html('');
+  weaponsData.getAllWeapons().then((response) => {
+    if (response.length) {
+      response.forEach((item) => {
+        $('#app').append(card.weaponsCard(item));
+      });
+    } else {
+      $('#app').append('<h2> NO CARDS!</h2>');
+    }
+    $('.delete').on('click', (e) => {
+      e.stopImmediatePropagation();
+      /* eslint-disable */
```

Comment:
Still am not liking this. What are we doing here? Let's set the ID to `e.currentTarget.id.split('__')[1]` if necessary. You are not using `type` anywhere that I see

Reply:
Fair! I'll take care of it!

Reply:
Done!